### DISCUSSION TITLE
Custom Columns

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#443] [Ideas]
Hi Team,

Not everyone wants to see the same set of columns in the backend. 

For example I want to see Units/Price and Stock Management Status. I don't see the option to configure. It will be better if you allow us to configure the column in the backend of our choice like most other CMS.

![image](https://user-images.githubusercontent.com/16856248/147816701-3aaa8092-3f75-43a2-9c3a-5ed698969f9b.png)

**Comment by Blair2004** [comment]
Nice suggestion.
